main.py

```
@@ -0,0 +1,479 @@
+                "nickname": user_data.nickname,
+                "profile_image": user_data.profile_image,
+                "created_at": current_time
+            }
+            }
+
+
+# 프로필 수정
+@app.put("/users/me", response_model=user.UpdateUserResponse)
+async def put_user(
+        update_data: Annotated[user.UpdateUserRequest, Body()],
+        authorization: Annotated[str, Header(description="로그인 시 발급받은 토큰")]
```

Ruff (0.14.13)

Unused function argument: `authorization`

(ARG001)